.github/actions/step_publish_test_results/action.yml

```diff
@@ -56,17 +56,17 @@ runs:
       indicators: true
       output: both
       thresholds: '${{ inputs.coverageThreshold }} 80'
-  # - name: Sending PR status to checkenforcer
-  #   run: |-
-  #     gh api \
-  #       --method POST \
-  #       -H "Accept: application/vnd.github+json" \
-  #       -H "X-GitHub-Api-Version: 2022-11-28" \
-  #       /repos/${{ github.repository }}/statuses/${{ github.sha }} \
-  #       -f state='success' \
-  #       -f target_url='https://github.com/microsoft/promptflow/actions/runs/${{ github.run_id }}' \
-  #       -f description='The build succeeded!' \
-  #       -f context='${{ inputs.context }}'
-  #   shell: bash -el {0}
-  #   env:
-  #     GITHUB_TOKEN: ${{ inputs.token }}
+  - name: Sending PR status to checkenforcer
```

Comment:
IIRC, `pull_request` cannot do this?

Reply:
let me remove this change, I copied this from original PR